app/ui/client/views/app/room.js

```diff
@@ -554,10 +554,12 @@ Meteor.startup(() => {
 				addClass.forEach((message) => $(`.messages-box #${message}`).addClass('selected'));
 			}
 		},
-		'click .jump-recent button'(e, template) {
+		'click .jump-recent button'(e) {
 			e.preventDefault();
-			template.atBottom = true;
-			RoomHistoryManager.clear(template && template.data && template.data._id);
+			const wrapperObj = document.getElementsByClassName('wrapper')[0];
+			wrapperObj.style.scrollBehavior = 'smooth';
+			const { scrollHeight, clientHeight } = wrapperObj;
+			wrapperObj.scrollTop = scrollHeight - clientHeight;
```

Comment:
It doesn't work well when messages have to be loaded from server while going to the bottom.

Reply:
Hey, @tassoevan Can you check now, added a smooth scroll inside sentToBottom.